datasets/doc/source/how-to-use-with-local-data.rst

```diff
@@ -0,0 +1,226 @@
+Use with Local Data
+===================
+
+You can partition data from local files and in-memory data in
+``Flower Datasets`` library using ``Partitioner`` s.
```

Comment:
```suggestion
You can create ``Flower Datasets`` from your local files or in-memory
data structures and then partition them using a built-in ``Partitioner``.
```

Reply:
We also show how to create the datasets, so I though of highlighting it right from the start.

Reply:
They can't create FederatedDatasets. And Flower Datasets is a name of the library, so I'd not use it in that context. I'll think about how to rewrite it.

Reply:
*a HF Dataset (maybe worth highlighting that's what's under-the-hood in FlowerDatasets?)